diff -ruN emu-tools-0.9.4aa/Makefile.config emu-tools-0.9.4/Makefile.config
--- emu-tools-0.9.4aa/Makefile.config	Sun Jan 20 03:00:17 2002
+++ emu-tools-0.9.4/Makefile.config	Tue Jan 29 18:39:47 2002
@@ -1,9 +1,9 @@
 
 #tools install defaults
 
-prefix ?= ${DESTDIR}/usr/local
+prefix := ${DESTDIR}/usr
 exec_prefix := $(prefix)
 bindir := $(exec_prefix)/bin
 data_dir := $(prefix)/share/emu10k1
-man_prefix := $(exec_prefix)/man
-script_dir := $(prefix)/etc
+man_prefix := $(prefix)/share/man
+script_dir := $(DESTDIR)/etc
diff -ruN emu-tools-0.9.4aa/epache-0.1.4/main.h emu-tools-0.9.4/epache-0.1.4/main.h
--- emu-tools-0.9.4aa/epache-0.1.4/main.h	Wed Oct 24 06:14:28 2001
+++ emu-tools-0.9.4/epache-0.1.4/main.h	Tue Jan 29 18:40:00 2002
@@ -25,7 +25,7 @@
 #define DSPMGR "emu-dspmgr"
 #define MAXFILENAME 32
 #define MAXPATHLEN 4095
-#define DIRWITHPATCHES "/usr/local/share/emu10k1/"
+#define DIRWITHPATCHES "/usr/share/emu10k1/"
 
 #define g_free(ptr) if (ptr) {g_free(ptr);ptr=0;}
 
diff -ruN emu-tools-0.9.4aa/scripts/Makefile emu-tools-0.9.4/scripts/Makefile
--- emu-tools-0.9.4aa/scripts/Makefile	Sun Jan 20 03:00:16 2002
+++ emu-tools-0.9.4/scripts/Makefile	Tue Jan 29 18:39:47 2002
@@ -8,5 +8,5 @@
 else
 	@echo "configuration file already installed..."
 endif
-	install -m 755 emu-script $(script_dir)
+	install -m 755 emu-script $(bindir)
 
diff -ruN emu-tools-0.9.4aa/scripts/emu-script emu-tools-0.9.4/scripts/emu-script
--- emu-tools-0.9.4aa/scripts/emu-script	Sun Jan 20 03:00:16 2002
+++ emu-tools-0.9.4/scripts/emu-script	Tue Jan 29 18:39:47 2002
@@ -50,7 +50,7 @@
 # unset these on the fly being particularly useful.
 
 # Default location of programs: 
-BASE_PATH=/usr/local
+BASE_PATH=/usr
 DSPPATH=$BASE_PATH/share/emu10k1
 
 DSPMGR=$BASE_PATH/bin/emu-dspmgr
@@ -63,7 +63,7 @@
 load(){
 
 # Source configurations
-  . $BASE_PATH/etc/emu10k1.conf
+  . /etc/emu10k1.conf
 
 # Pick up any coomand line overrides
 while getopts d:t:i:r:m:s:b: OPT $SAVEARGS; do
